### PR TITLE
Split 13914 mapping the e2e test scripts

### PR DESCRIPTION
**Pull request by xiuwang**
Passed in the local.

### CODE REVIEW

File: features/build/buildlogic.feature [hunk 1]
```diff
@@ -258,65 +258,173 @@ Feature: buildlogic.feature
...
+    When I run the :new_build client command with:
+      | app_repo | https://github.com/openshift/ruby-hello-world.git |
+    Then the step should succeed
+    When I run the :describe client command with:
+      | resource | buildconfig      |
+      | name     | ruby-hello-world |
+    Then the step should succeed
+    Then the output should contain:
+      | Builds History Limit |
+      | Successful:	5    |
```
Comment (wewang58) [inline features/build/buildlogic.feature]: better to align

Comment (wewang58) [reply features/build/buildlogic.feature]: other lgtm

File: features/build/buildlogic.feature [hunk 2]
```diff
@@ -258,65 +258,173 @@ Feature: buildlogic.feature
...
+      | Git.*Cancelled |
     Then the output should not contain:
-      |ruby-hello-world-1|
+      | rails-postgresql-example-1 |
     """
-    Given I run the steps 3 times:
+
+  # @author xiuwang@redhat.com
+  # @case_id OCP-24155
+  # Split 13914 mapping the e2e test scripts
```
Comment (akostadinov) [inline features/build/buildlogic.feature]: Do we need this comment and other similar ones? After the split we have separate test cases and nobody would care what the original one was. Or not?

Comment (akostadinov) [reply features/build/buildlogic.feature]: otherwise lg, thanks

Comment (xiuwang) [reply features/build/buildlogic.feature]: Right, nobody cares the original one. I will remove these comments

Comment (xiuwang) [reply features/build/buildlogic.feature]: Only keep comment for the original case OCP-13914.

**Comment by wzheng1** [comment]
LGTM, thanks!

**Comment by akostadinov** [comment]
I believe commit message is enough for anynody that wants to understand the change. So I removed last comment and I'm merging. Let me know if comment is important to you.

Thank you!

**Pull request merged**